src/Engines/constants.ts

```diff
@@ -594,4 +594,10 @@ export class Constants {
      * Constant used as key code for Shift key
      */
     public static readonly INPUT_SHIFT_KEY = 16;
+
+    /** Standard snapshot rendering. In this mode, some form of dynamic behavior is possible (for eg, uniform buffers are still updated) */
+    public static readonly SNAPSHOTRENDERING_STANDARD = 0;
+
+    /** Fast snapshot rendering. In this mode, everything is static and only some limited form of dynamic behaviour is possible */
+    public static readonly SNAPSHOTRENDERING_FAST = 1;

```

Comment:
will this be enabled when users call scene.frezzeActiveMeshes?

Reply:
No, it's independent from freezeActiveMeshes. It's activated when setting `engine.snapshotRendering = true`.

Reply:
do you think it is an option to do i? Does it make sense?

Reply:
No, I think it's really two different things, you could want freezing active meshes without enabling snapshot rendering as snapshot rendering have more constraints than freezing meshes (meaning, when snapshot rendering is enabled a number of things won't work as expected if you do them that would work if you only froze active meshes).